pkg/apis/kubermatic/v1/settings.go

```diff
@@ -249,6 +252,21 @@ type StaticLabel struct {
 	Protected bool     `json:"protected"`
 }
 
+// The announcement feature allows administrators to broadcast important messages to all users.
+type Announcement struct {
+	// The message content of the announcement.
+	Message string `json:"message"`
+	// Unique identifier for the announcement.
+	ID string `json:"id"`
+	// Represent the status of the announcement.
+	Status string `json:"status"`
```

Comment:
What are the possible status values for an announcement? Can we maybe define an ENUM for this? Should we maybe also add a +default thingy to set it to some default value if left empty?

Reply:
Actually, it's either active or not, so I changed it to a boolean type.